src/test_utils/katana/mod.rs

```diff
@@ -48,27 +49,22 @@ fn load_genesis() -> Genesis {
     .expect("Failed to convert GenesisJson to Genesis")
 }
 
-/// Returns a `StarknetConfig` instance customized for Kakarot.
-/// If `with_dumped_state` is true, the config will be initialized with the dumped state.
-pub fn katana_config() -> StarknetConfig {
-    let max_steps = u32::MAX;
-    StarknetConfig {
-        disable_fee: true,
-        env: Environment {
-            // Since kaka_test > u32::MAX, we should return the last 4 bytes of the chain_id: test
-            chain_id: ChainId::parse("test").unwrap(),
-            invoke_max_steps: max_steps,
-            validate_max_steps: max_steps,
-        },
-        genesis: load_genesis(),
-        ..Default::default()
-    }
-}
-
 /// Returns a `TestSequencer` configured for Kakarot.
 #[cfg(any(test, feature = "arbitrary", feature = "testing"))]
 pub async fn katana_sequencer() -> TestSequencer {
-    TestSequencer::start(SequencerConfig { no_mining: false, block_time: None }, katana_config()).await
+    TestSequencer::start(Config {
+        chain: ChainSpec {
+            // Since kaka_test > u32::MAX, we should return the last 4 bytes of the chain_id: test
```

Comment:
I think we can remove this comment or reuse kaka_test as chain id

Reply:
yes true should be, just fixed